appveyor.yml

```diff
@@ -48,7 +46,6 @@ install:
       }
   - cmd: SET M2_HOME=C:\maven\apache-maven-3.2.5
   - cmd: SET PATH=%M2_HOME%\bin;%JAVA_HOME%\bin;%PATH%
-  - cmd: git config --global core.autocrlf
```

Comment:
lets restore this line, it just print value of config, to be sure in what mode code is checkouted.
We do not need "--global" as we chekouted repo config to be printed, irrespective of what is global in appveyor.